packages/artifact/src/internal/utils.ts

```diff
@@ -202,6 +204,9 @@ export function getUploadHeaders(
   if (contentRange) {
     requestOptions['Content-Range'] = contentRange
   }
+  if (digest) {
+    requestOptions['X-GH-Actions-CRC64'] = digest
```

Comment:
I came up with `X-GH-Actions-CRC64` for the header, willing to change if there's a better alternative 👍

Reply:
For normal parts of the site I see stuff like this:

![image](https://user-images.githubusercontent.com/16109154/165627316-0b79ebda-1767-4888-b038-5b7365062bab.png)

so how about something like `x-gh-actions-crc64` for consistency with lowercase? I'm also leaning towards `x-actions-result-crc64` (with our future architecture this might make sense)

Reply:
I like `x-actions-result-crc64` 👍 Also HTTP Headers are case insensitive so we shouldn't need to worry about that